src/index.js

```diff
@@ -0,0 +1,62 @@
+import './index.css';
+import getProducts from './modules/getProducts.js';
+import itemPopup from './modules/itemPopup.js';
+import closePopup from './modules/closePopUp.js';
+import getResFormData from './modules/getResFormData.js';
+
+import TheLikes from './modules/showLikes.js';
+
+import productCount from './modules/productCount.js';
+
+const store = document.querySelector('.counter');
+
+const logo = document.querySelector('.logo');
+logo.innerHTML = '<div class="oke">O K E</div>';
+
+// Items Interface
+const itemsContainer = async () => {
+  const likes = await new TheLikes().likesStoredData;
+  const theProducts = await getProducts();
+
+  const productsList = document.getElementById('products-list');
+  productsList.innerHTML = '';
+  const getAllProducts = async () => {
+    for (let i = 0; i < theProducts.length; i += 1) {
+      const itemId = i + 1;
+      productsList.insertAdjacentHTML(
+        'beforeend',
+        `
+        <li class="store-items">
+        <img class="store-items-img" src="${theProducts[i].image}"/>
+        <div class="title-like"><h2 class="itemid">Item ${itemId} </h2>
+        <i id="${i}" class="fa fa-heart likebtn" aria-hidden="true"></i>
+        <p class="likes-count">${likes[i].likes}</p>
```

Comment:
:red_circle: [REQUIRED] There's an Uncaught runtime error: "likes is null" in this code. Team, please investigate and fix this issue to ensure proper error handling.
![erro](https://github.com/ometman/JS-capstone-kanban/assets/26736582/96b1014c-42c4-498c-8ff9-7acd1ad6b381)

Reply:
Thanks but I noticed it is working fine locally